<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMSISetupClass</class>
 <widget class="QWidget" name="TMSISetupClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TMSi EEG Connector Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="m_qLabel_Headline">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TMSi EEG Connector</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="m_qVerticalSpacer_Headline">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="m_qGridLayout_main">
     <item row="4" column="0">
      <spacer name="m_qVerticalSpacer_LeftRow">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="m_qGroupBox_SamplingProperties">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Device sampling properties</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="m_qGridLayout_Properties">
        <item row="2" column="1">
         <widget class="QSpinBox" name="m_spinBox_SamplesPerBlock">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>512</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="m_qLabel_SamplingRate">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Sampling Frequency (Hz):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="m_spinBox_SamplingFreq">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>2048</number>
          </property>
          <property name="value">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="m_spinBox_NumberOfChannels">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>138</number>
          </property>
          <property name="value">
           <number>138</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="m_qLabel_NumberOfChannels">
          <property name="text">
           <string>Number of channels</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="m_qLabel_SamplesPerBlock">
          <property name="text">
           <string>Samples taken per block</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="m_qPushButton_About">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="m_qHorizontalSpacer_About">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" rowspan="5" colspan="2">
      <widget class="QGroupBox" name="m_qGroupBox_Information">
       <property name="title">
        <string>Information</string>
       </property>
       <layout class="QGridLayout" name="m_qGridLayout_Information">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="m_qTextBrowser_Information"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="m_qGroupBox_ChannelCorrections">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Channel corrections</string>
       </property>
       <layout class="QGridLayout" name="m_qGridLayout_Channels">
        <item row="5" column="0">
         <widget class="QCheckBox" name="m_checkBox_UseUnitOffset">
          <property name="text">
           <string>Use unit offset</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="m_checkBox_UseUnitGain">
          <property name="text">
           <string>Use unit gain</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="m_checkBox_ConvertToVolt">
          <property name="text">
           <string>Convert to Volt</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="m_checkBox_UseChExponent">
          <property name="text">
           <string>Use channel's exponent</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="m_qGroupBox_WriteToFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Write received samples to file</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QCheckBox" name="m_checkBox_WriteToFile">
          <property name="text">
           <string>Write samples to file</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="m_lineEdit_outputDir">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>/mne_x_plugins/resources/tmsi</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="m_pushButton_ChangeDir">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Change dir</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Current output dir:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
